source/guide_phpservermonitor.rst

```diff
@@ -0,0 +1,114 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/phpservermonitor.png
+      :align: center
+
+##################
+PHP Server Monitor
+##################
+
+.. tag_list::
+
+PHP Server Monitor_ is an open source tool to monitor your servers and websites.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.2'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install PHP Server Monitor we download the current version from the github release. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the zip file will be under your ``html``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/phpservermon/phpservermon/releases/download/v3.4.5/phpservermon-3.4.5.zip
+ [isabell@stardust html]$ unzip phpservermon-3.4.5.zip
+ [isabell@stardust html]$ rm phpservermon-3.4.5.zip
+ [isabell@stardust html]$ mv phpservermon-3.4.5/* phpservermon-3.4.5/.* ./
+ [isabell@stardust html]$ rmdir phpservermon-3.4.5
+ [isabell@stardust html]$ 
+
+Configuration
+=============
+
+Create the database, copy the configuration template and edit the ``config.php`` file with your SQL credentials.
+
+.. code-block:: console
+
+ [isabell@stardust html]$ mysql -e "CREATE DATABASE ${USER}_psm"
+ [isabell@stardust html]$ cp config.php.sample config.php
+ [isabell@stardust html]$
+
+After the installation you need to open isabell.uber.space in your browser to finish your setup.
+
+Fill out your system settings, admin user and edit the following database settings:
+ * PSM_DB_USER: ``isabell``
+ * PSM_DB_PASS from your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>`
+ * PSM_DB_NAME: ``isabell_psm``
+
+::
+
+Finally, configure a cronjob: Add the following line to your crontab using the crontab -e command:
+
+* * * * * /usr/bin/php /var/www/virtual/isabell/html/cron/status.cron.php
```

Comment:
The `::` from line 79 has to move down a bit [to fix the formatting](https://deploy-preview-571--uberlab.netlify.com/guide_phpservermonitor.html#configuration).